docs/tutorials/patterns.md

```diff
@@ -283,3 +283,84 @@ Webhook GitHub:
 Finally, the GitHub WebHook payload url is the url above with `<secret>`
 replaced with base64 decoded string of the value of `data.WebHookSecretKey`
 above and the content type is `application/json`.
+
+## Custom domain names and secure transport
+
+Custom domain names and HTTPS secure data transport are implemented in the
+Route object level. They are controlled with the keywords `spec.host` and
+`spec.tls`.
+
+The public DNS entry of the custom domain name should point to public IP of the
+domain `rahtiapp.fi` (currently `193.167.189.101`) and the custom DNS name is
+placed in the `spec.host` entry of the Route object:
+
+*`route-with-dns.yaml`*:
+
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  labels:
+    app: serveapp
+  name: myservice
+spec:
+  host: my-custom-dns-name.replace.this.com
+  to:
+    kind: Service
+    name: serve
+    weight: 100
+```
+
+The TLS certificates and private keys are placed in the `spec.tls` field, for
+example:
+
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  labels:
+    app: serveapp
+  name: myservice
+spec:
+  host: my-custom-dns-name.replace.this.com
+  to:
+    kind: Service
+    name: serve
+    weight: 100
+  tls:
```

Comment:
Here I would arrange the fields in non-alphabetical order and keep the conceptually close stuff together, maybe termination and redirect policy first and then the cert+key data?